MINOR: PartitionInfo toString corrected

kamalcph opened the pull request:
- Removed the extra ',' character while printing the replicas / in-sync replicas identifier

Commit:
MINOR: PartitionInfo toString corrected

- Removed the extra ',' character while printing the replicas / in-sync replicas identifier

ijuma commented:
Thanks for the PR. Can you please add a unit test? Since the original bug eluded us, it indicates that a simple test that validates the output would be useful.

ijuma commented on clients/src/main/java/org/apache/kafka/common/PartitionInfo.java:
```diff
@@ -80,13 +80,10 @@ public String toString() {
     /* Extract the node ids from each item in the array and format for display */
     private String fmtNodeIds(Node[] nodes) {
```
Can you please rename this to `formatNodeIds`. The name is unnecessarily cryptic (unrelated to this PR, I know).

Commit:
MINOR: PartitionInfo toString corrected

- Removed the extra ',' character while printing the replicas / in-sync replicas identifier
- PartitionInfoTest added for `toString` method.
- Renamed the method 'fmtNodeIds' to 'formatNodeIds'

kamalcph commented:
@ijuma

Updated the method name and unit test added.

kamalcph committed:
Imports corrected

ijuma commented:
LGTM, thanks. Merged to trunk.
